OpenRA.Mods.Common/Widgets/Logic/Ingame/ClassicProductionLogic.cs

```diff
@@ -78,6 +78,7 @@ public ClassicProductionLogic(Widget widget, OrderManager orderManager, World wo
 				if (foreground != null)
 					foregroundTemplate = foreground.Get("ROW_TEMPLATE");
 
+				var powerPeter = widget.GetOrNull<SpritePowerMeterWidget>("POWERMETER");
```

Comment:
`powerMeter` with an M! This probably also wants to be called `POWER_METER` or maybe `POWER_BAR` in yaml to match the other names.